src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/model/APIListing.java

```diff
@@ -23,6 +23,9 @@ public class APIListing {
     @JsonProperty("Language")
     private String language;
 
+    @JsonProperty("Variant")
```

Comment:
nit: Let's name the property `LanguageVariant` to match the type and is more specific than just `variant`. You might want to consider doing that in other places too.